src/SwedbankPay.Sdk.Tests/TestHelpers/FakeDelegatingHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Tests.TestHelpers
+{
+    public class FakeDelegatingHandler : DelegatingHandler
+    {
+        public List<HttpResponseMessage> FakeResponseList { get; } = new List<HttpResponseMessage>();
+        public List<HttpRequestMessage> RequestMessageList { get; } = new List<HttpRequestMessage>();
+
+        protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+        {
+            RequestMessageList.Add(request);
+            var fakeResponse = FakeResponseList[0];
+            FakeResponseList.RemoveAt(0);
+            return await Task.FromResult(fakeResponse);

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5363487)